docs/faq/why-linkml.md

```diff
@@ -277,16 +277,16 @@ See also:
 
 UML is a powerful language for software engineer diagramming. [UML
 Class Diagrams](https://en.wikipedia.org/wiki/Class_diagram) provide a
-standard way to draw a class hierarchy for a program, that has some
+standard way to draw a class hierarchy for a program that has some
 similarities to LinkML.
 
-There are a number of reasons to use LinkML over UML
+There are a number of reasons to use LinkML over UML:
 
 * The UML standard is large and complex
-* UML serialization in XMI is hard to work with
+* UML serialization in XML is hard to work with
```

Comment:
this wasn't a typo. XML Metadata Interchange: https://en.wikipedia.org/wiki/XML_Metadata_Interchange

Reply:
oh, oops. i thought Mark had flagged it as a typo in #2081. i will fix.